.github/workflows/django-overhead-profile.yml

```
@@ -0,0 +1,92 @@
+name: django-overhead-profile
+on:
+  push:
+    branches: [master]
+  pull_request:
+jobs:
+  django-overhead-profile:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: trace-examples/python/django/django-simple
+    env:
+      DJANGO_ALLOWED_HOSTS: 127.0.0.1
+      DJANGO_SECRET_KEY: vNIF0IeM1IN9YAOOB8yEmNddSvnnXxx7E7D3TwWsdno
```

can we make this obviously not real
```suggestion
      DJANGO_SECRET_KEY: SECRET_KEY
```

or something?

---

The k6 stuff can be taken out as this PR focuses on profiling.